lib/core/controlplane.rb

```diff
@@ -23,6 +23,7 @@ def profile_exists?(profile)
   end
 
   def profile_create(profile, token)
+    @sensitive_data_pattern = /(?<=--token )(\S+)/
```

Comment:
This way, we can customize the sensitive_data_pattern for each method based on what we add to them.

Are we sure we won't have any other cases of sensitive data?

Reply:
I mean, if we want to add more patterns later on we can just extend the regex. Having it in a single place (probably in the `Shell` class, since it implements `debug`) would make it easier to update later on, and easier to find all patterns.